src/app/tests/suites/DL_UsersAndCredentials.yaml

```diff
@@ -1327,6 +1520,83 @@ tests:
               - name: "NextCredentialIndex"
                 value: 2
 
+    - label:
+          "Attempt to create new RFID credential from different fabric and add
+          it to existing user"
+      command: "SetCredential"
+      identity: "beta"
+      timedInteractionTimeoutMs: 10000
+      arguments:
+          values:
+              - name: "OperationType"
+                value: 0
+              - name: "Credential"
+                value: { CredentialType: 2, CredentialIndex: 1 }
+              - name: "CredentialData"
+                value: "rfid_data_123456"
+              - name: "UserIndex"
+                value: 1
+              - name: "UserStatus"
+                value: null
+              - name: "UserType"
+                value: null
+      response:
+          values:
+              - name: "Status"
+                value: 0x85
```

Comment:
Can't this use INVALID_COMMAND as a name?  Similar in other places.

If not, that's unfortunate and worth an issue.